web/cypress/integration/lang/switch-language.spec.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* eslint-disable no-undef */
+
+context('Switch language', () => {
+  const timeout = 1000;
+  const domSelector = {
+    switcher: '.ant-space-align-center',
+  };
+
+  beforeEach(() => {
+    cy.login();
+  });
+
+  it('should switch language', () => {
+    cy.visit('/');
+
+    cy.get(domSelector.switcher).click('right');
+    cy.contains('简体中文').click({
+      timeout,
+    });
+    cy.contains('服务').click();
+
+    cy.get(domSelector.switcher).click('right');
+    cy.contains('English').click({
+      timeout,
+    });
+    cy.contains('Create');
```

Comment:
does it need `.should('exist')` here?

Reply:
`.should("contain", "TEXT")`?

Reply:
@LiteSun Please take a look at here.

Reply:
You cannot use .should() here, as shown in the figure: 
![image](https://user-images.githubusercontent.com/72343596/105151529-092edc80-5b41-11eb-9e12-74c5d4ffaf2c.png)